src/App.jsx

```diff
@@ -6,43 +6,44 @@ import ItemViewPage from 'components/Items/ItemViewPage';
 import LoginPage from 'components/Login/LoginPage';
 import RegisterPage from 'components/Login/RegisterPage';
 import LandingPage from 'views/LandingPage/LandingPage';
-import ItemListings from 'components/Items/ItemListings';
+import ItemListPage from 'components/Items/ItemListPage';
 
 import Profile from './components/Profile/Profile';
 import Forgotten from './components/Login/Forgotten';
 import ResetPassword from './components/Login/ResetPassword';
 import AddEditItem from './components/Items/AddEditItem';
+import ItemListings from 'components/Items/ItemListings';
 
 function App() {
   const [data, setData] = useState(TestData);
 
   return (
     <BrowserRouter>
       <Switch>
-        <Route path="/login" component={LoginPage} />
-        <Route path="/register">
+        <Route path='/login' component={LoginPage} />
```

Comment:
These should all be double quotes within the jsx according to the airbnb rules in eslint, you may have to update your prettier settings to use double quotes in jsx.